src/routes/+page.svelte

```diff
@@ -3,7 +3,7 @@
 
     import LandingHeader from "./LandingHeader.svelte";
     import LinkButton from "./LinkButton.svelte";
-
+    import Signin from "$lib/components/Signin.svelte";
```

Comment:
Can you name the component `SignIn` (i.e. capitalise "in")

Reply:
Sure yeah